mlmpfr.opam

```diff
@@ -8,35 +8,25 @@ bug-reports:  "https://github.com/thvnx/mlmpfr/issues"
 license:      "LGPL-3.0"
 dev-repo: "git+https://github.com/thvnx/mlmpfr.git"
 build: [
-  ["cc" "test.c" "-lmpfr" "-o" "test_installed"]
-  ["./test_installed"]
-  ["oasis" "setup"]
-  ["./configure" "--prefix=%{prefix}%"]
-  [make]
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}]
 ]
-install: [make "install"]
-remove:  ["ocamlfind" "remove" "mlmpfr"]
 depends: [
   "ocaml" {>= "4.04"}
-  "ocamlfind"
-  "oasis"
+  "dune" {>= "1.11.0"}
 ]
 depexts: [
-  ["libmpfr-dev"] {os-distribution = "ubuntu"}
-  ["libmpfr-dev"] {os-distribution = "debian"}
+  ["libmpfr-dev"] {os-family = "debian"}
 ]
 post-messages: [
   "Make sure you had MPFR version 4.0.2 installed on your system." {failure}
 ]
 synopsis: "OCaml C bindings for MPFR-4.0.2"
-flags: light-uninstall
-url {
```

Comment:
Maybe this is more opam-related stuff, but I do not understand how it works without the url/tar.gz. What happens if you request an installation via opam? Is the master branch the new reference?

Reply:
The light-uninstall is useless now IIRC.
For the url part it's just that it's common practice not to have it in your dev repo and to only add it when sending it to opam. Note that you have [dune-release](https://github.com/samoht/dune-release) that can take care of almost everything for you. :)

Reply:
I see, thanks!